app/assets/javascripts/modal/modal.controller.js

```diff
@@ -11,6 +11,34 @@ function ModalController($rootScope, DocumentService, $scope, UserService){
     $ctrl.docs = [];
     $ctrl.book = {};
 
+    $ctrl.fuzzySearchString = '';
+
+    $ctrl.fuzzySearchList = [];
+
+    $ctrl.fuzzySearchOptions = {
+        shouldSort: true,
+        threshold: 0.6,
+        location: 0,
+        distance: 30,
+        maxPatternLength: 32,
+        minMatchCharLength: 1,
+        keys: [
+            "name",
```

Comment:
should this be `filename` as documents have a filename attribute as opposed to `name` attribute?

Reply:
Absolutely correct